batchflow/models/tf/efficientnet.py

```diff
@@ -0,0 +1,242 @@
+""" EfficientNet family of NNs
+
+Mingxing Tan, Quoc V. Le "`EfficientNet: Rethinking Model Scaling for Convolutional Neural Networks
+<https://arxiv.org/abs/1905.11946>`_"
+
+"""
+import tensorflow as tf
+
+from . import TFModel, MobileNet_v2, EncoderDecoder
+
+
+class EfficientNetB0(EncoderDecoder):
+    """
+    EfficientNetB0
+
+    Parameters
+    ----------
+    inputs : dict
+        dict with 'images' and 'labels' (see :meth:`~tf.TFModel._make_inputs`)
+
+    initial_block : dict, optional
+        parameters for the initial block (see :class:`~tf.EncoderDecoder`)
+        scalable : bool
+            indicates whether the block can be scaled
+
+    body : dict, optional
+        encoder : dict, optional
+            parameters for model's body (see :class:`~tf.EncoderDecoder`)
+
+            num_stages : int, optional
+                Number of blocks
```

Comment:
Some of the docstrings are formatted in such a way that every description starts with a capital letter and ends with a point, some of them don't. Choose one:)